lib/node_modules/@stdlib/math/base/special/gammasgn/package.json

```
@@ -0,0 +1,63 @@
+{
+  "name": "@stdlib/math/base/special/gammasgn",
+  "version": "0.0.0",
+  "description": "Sign of the gamma function",
```

```suggestion
  "description": "Sign of the gamma function.",
```

Descriptions in `package.json` files always end in a period.